### ISSUE TITLE
[Typo]: useActionState reference page doesn't include `isPending` return, and is only used with forms

### ISSUE THREAD

**Issue by poulet42** [reactjs/react.dev#6872]
### Summary

Hello 👋 
1. useActionState should return `[state, formAction, isPending]`, not just `[state, formAction]`
2. The doc make it unclear if we can actually use this hook **outside** of a form, there are only examples about using it in a form and the second item in the return is `formAction`, but [the author of the hook specified it should not always be the case](https://github.com/facebook/react/pull/28491)

### Page

https://19.react.dev/reference/react/useActionState#returns

**Comment by ijayhub** [comment]
can I work on this if it valid

**Comment by vaibhavibaware** [comment]
This issue appears to have been resolved in a recent update .The relevant file now contains the necessary documentation and examples.

**Comment by rickhanlonii** [comment]
Yeah we really need to update this, I think it was just copied over from when it was `useFormState` and only used with forms.
